frontend/src/pages/Projects/Project/ProjectDetails/AgreementSection/index.tsx

```diff
@@ -1,74 +1,108 @@
-import { Box, Button } from "@mui/material";
-import { EditForm } from "components/EditForm";
-import { ReadForm } from "components/ReadForm";
-import { Renderer } from "components/Renderer";
-import { useFormSubmit } from "hooks/useFormSubmit";
-import React, { useEffect, useState } from "react";
-import { useParams } from "react-router-dom";
-import { editFields, readFields } from "./fields";
+// import { Box, Button } from "@mui/material";
+// import { EditForm } from "components/EditForm";
+// import { ReadForm } from "components/ReadForm";
+// import { Renderer } from "components/Renderer";
+// import { useFormSubmit } from "hooks/useFormSubmit";
+// import React, { useEffect, useState } from "react";
+// import { useParams } from "react-router-dom";
+// import { editFields, readFields } from "./formFields";
 
-export const AgreementSection = ({
-  query,
-}: {
-  /* eslint "no-warning-comments": [1, { "terms": ["todo", "fixme"] }] */
-  // todo Define a good type. "Any" type temporarily permitted.
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  query: any;
-}) => {
-  const { projectId } = useParams();
-  const [editMode, setEditMode] = useState(false);
-  const { handleUpdate, Notification } = useFormSubmit();
+// export const AgreementSection = ({
```

Comment:
is there a reason for leaving in the commented code?  In future, if we do this, then we should indicate the intention.  If it is no longer required, remove it, if you think you might need it, why not add a comment with the previous commit, so you have reference.